onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -57,73 +235,42 @@ def quantize_static(
-            ForceQuantizeNoInputCheck = True/False : By default, some latent operators like maxpool, transpose, do not quantize
-                                                     if their input is not quantized already. Setting to True to force such operator
-                                                     always quantize input and so generate quantized output. Also the True behavior
-                                                     could be disabled per node using the nodes_to_exclude.
-            MatMulConstBOnly = True/False: Default is False for static mode. If enabled, only MatMul with const B will be quantized.
-            AddQDQPairToWeight = True/False : Default is False which quantizes floating-point weight and feeds it to
-                                              soley inserted DeQuantizeLinear node. If True, it remains floating-point weight and
-                                              inserts both QuantizeLinear/DeQuantizeLinear nodes to weight.
-            OpTypesToExcludeOutputQuantizatioin = list of op type : Default is []. If any op type is specified, it won't quantize
-                                                                    the output of ops with this specific op types.
-            DedicatedQDQPair = True/False : Default is False. When inserting QDQ pair, multiple nodes can share a single QDQ pair as their inputs.
-                                            If True, it will create identical and dedicated QDQ pair for each node.
-            QDQOpTypePerChannelSupportToAxis = dictionary : Default is {}. Set channel axis for specific op type, for example: {'MatMul': 1},
-                                                            and it's effective only when per channel quantization is supported and per_channel is True.
-                                                            If specific op type supports per channel quantization but not explicitly specified with channel axis,
-                                                            default channel axis will be used.
-            CalibTensorRangeSymmetric = True/False : Default is False. If enabled, the final range of tensor during calibration will be explicitly set to symmetric to central point "0".
-            CalibMovingAverage = True/False : Default is False. If enabled, the moving average of the minimum and maximum values
-                                              will be computed when the calibration method selected is MinMax.
-            CalibMovingAverageConstant = float : Default is 0.01. Constant smoothing factor to use when computing the moving average of
-                                                 the minimum and maximum values. Effective only when the calibration method selected is
-                                                 MinMax and when CalibMovingAverage is set to True.
+
     """
+    if "quant_config" in kwargs:
+        qc = kwargs["quant_config"]
+
+    else:
+        qc = StaticQuantConfig(
```

Comment:
naming: abbreviation: qc. I would write `static_quant_config` or `quant_config` instead

readability: Avoid abbreviations - they add cognitive load to readers.

> Do not use abbreviations that are ambiguous or unfamiliar to readers outside the project, and do not abbreviate by deleting letters within a word.

https://google.github.io/styleguide/pyguide.html#316-naming
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules